chatwoot/chatwoot
# fix vertical overflow scroll on `BackButton`

Fixes: https://github.com/chatwoot/chatwoot/issues/6913

## Description

Removes `settings` class name from `BackButton` to prevent vertical scrollbar on Button

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Go to Teams
- Click on Create new teams
- Notice there's no scrollbar on the back button

## Checklist:


## Demo

<img width="1525" alt="Screenshot 2023-04-17 at 9 55 50 AM" src="https://user-images.githubusercontent.com/45232708/232435964-9efda4e5-876d-4b89-a1bd-f2fcf8b5fb82.png">
